### DISCUSSION TITLE
[Feature Request] Calendar 🗓️

### DISCUSSION THREAD

**Discussion by elguarir** [heroui-inc/heroui#1664] [Feature requests]
### Is your feature request related to a problem? Please describe.

calenders are one the most important components to have and hard to build, it would be nice to have, I've been using nextui for a long time now, and I love it, but each time I need to implement something that includes working with date pickers I have to relie on other libraries which is not ideal.
Thanks.

### Describe the solution you'd like

- normal date picker based on the react area ones
- calendar with time
- date range picker
- calendar with dropdown presets (if applicable)

### Describe alternatives you've considered

React DayPicker

**Comment by wingkwong** [comment]
see https://www.heroui.com/docs/components/calendar
